lib/view/components/events/joined_event_card.dart

```diff
@@ -92,26 +92,29 @@ class _UpcomingEventCardWidgetState extends State<UpcomingEventCardWidget> {
                             mainAxisAlignment: MainAxisAlignment.center,
                             crossAxisAlignment: CrossAxisAlignment.start,
                             children: [
-                              CachedNetworkImage(
-                                height: 83.65,
-                                width: 143,
-                                imageUrl: imageUrl,
-                                imageBuilder: (context, imageProvider) =>
-                                    Container(
-                                  decoration: BoxDecoration(
-                                    shape: BoxShape.rectangle,
-                                    borderRadius: BorderRadius.circular(12),
-                                    image: DecorationImage(
-                                        image: imageProvider,
-                                        fit: BoxFit.cover),
+                              Padding(
+                                padding: const EdgeInsets.fromLTRB(2, 0, 2, 6),
+                                child: CachedNetworkImage(
+                                  height: 83.65,
+                                  width: double.infinity,
+                                  imageUrl: imageUrl,
+                                  imageBuilder: (context, imageProvider) =>
+                                      Container(
+                                    decoration: BoxDecoration(
+                                      shape: BoxShape.rectangle,
+                                      borderRadius: BorderRadius.circular(12),
+                                      image: DecorationImage(
+                                          image: imageProvider,
+                                          fit: BoxFit.cover),
+                                    ),
                                   ),
+                                  placeholder: (context, url) => const Center(
+                                      child:
+                                          CircularProgressIndicator()), // Loading indicator for banner
```

Comment:
Looks good, could you change this to LoadingAnimationWidget.halfTriangleDot with size 36 please? Just so we move away from the old progress indicator?